docs/java/java-webapp.md

```diff
@@ -11,83 +11,74 @@ MetaDescription: Java web app tutorial showing how to deploy a Java web app to A
 
 In our [first tutorial](/docs/java/java-tutorial.md), we built a Java web application and ran it locally. In this tutorial, you will learn how to deploy and run it on [Azure](https://azure.microsoft.com) in the cloud.
 
-## Prepare your Azure account
+## Prerequisites
 
-Before running this sample application on [Azure](https://azure.microsoft.com), you need to have an Azure subscription and the Azure command line tools.
+To develop to Azure Web App with Java, you'll need the following tools installed:
```

Comment:
I think we can just go ahead to list the items without this line, what do you think? Maybe you could check some other popular open source projects for reference.